config.go

```diff
@@ -110,7 +109,7 @@ type config struct {
 	PruneTickets        bool    `long:"prunetickets" description:"Prune old tickets from the wallet and restore their inputs"`
 	PurchaseAccount     string  `long:"purchaseaccount" description:"Name of the account to buy tickets from (default: default)"`
 	TicketAddress       string  `long:"ticketaddress" description:"Send all ticket outputs to this address (P2PKH or P2SH only)"`
-	TicketMaxPrice      float64 `long:"ticketmaxprice" description:"The maximum price the user is willing to spend on buying a ticket (default: 100.0)"`
+	TicketMaxPrice      float64 `long:"ticketmaxprice" description:"The maximum price the user is willing to spend on buying a ticket. If less then 3, it acts as a multipler to the average price. (default: 1.25)"`
```

Comment:
Hmm not sure I like overloading this setting for TicketMaxPrice and TicketMaxPriceMultiplier, I'd seperate them to avoid confusion.  Plus the use of 3 as the cutoff is never explained.